src/components.js

```diff
@@ -0,0 +1,44 @@
+export const UserNameInput = () => (
+  <input
+    type="text"
+    className="user-name"
+    placeholder="Please enter user name"
+  ></input>
+);
+
+export const ChangeUserButton = () => <button>Set User</button>;
+
+export const NewTodoInput = () => (
+  <input
+    type="text"
+    className="new-todo"
+    placeholder="Please enter new todo here"
+  ></input>
+);
+
+export const Header = () => (
+  <header>
+    <h1>Todo App React</h1>
+  </header>
+);
+
+export const AddTodoButton = () => <button>Add todo</button>;
+
+export const FilterTodoListButton = () => (
+  <select>
+    <option value="all">All</option>
+    <option value="completed">Completed</option>
+    <option value="outstanding">Outstanding</option>
+  </select>
+);
+
+const Todo = () => (
+  <div className="todo-list">
+    <p>Todo 1</p>
+    <button className="complete-button">Complete</button>
+    <button className="edit-button">Edit</button>
+    <button className="delete-button">Delete</button>
+  </div>
+);
+
+export const TodoList = () => <Todo />;
```

Comment:
`<TodoList />` is a React component which in this case duplicates `<Todo />`

all related buttons should be separated out so that the structure looks more semantic e.g.

```js
const Todo = () => (
  <div className="todo-list">
    <p>Todo 1</p>
    <Complete />
    <Edit />
    <Delete />
  </div>
);
```